peripherals/spi/spi_v3.yaml

```diff
@@ -0,0 +1,281 @@
+      SevenFrames: [6, "7 frames"]
+      EightFrames: [7, "8 frames"]
+      NineFrames: [8, "9 frames"]
+      TenFrames: [9, "10 frames"]
+      ElevenFrames: [10, "11 frames"]
+      TwelveFrames: [11, "12 frames"]
+      ThirteenFrames: [12, "13 frames"]
+      FourteenFrames: [13, "14 frames"]
+      FifteenFrames: [14, "15 frames"]
+      SixteenFrames: [15, "16 frames"]
+    DSIZE: [3, 31]
+
+  CFG2:
+    AFCNTR:
+      NotControlled: [0, "Peripheral takes no control of GPIOs while disabled"]
+      Controlled: [1, "Peripheral controls GPIOs while disabled"]
+    SSOM:
+      Asserted: [0, "SS is asserted until data transfer complete"]
+      NotAasserted: [1, "Data frames interleaved with SS not asserted during MIDI"]
+    SSOE:
+      Disabled: [0, "SS output is disabled in master mode"]
+      Enabled: [1, "SS output is enabled in master mode"]
+    SSIOP:
+    SSM:
+      Disabled: [0, "Software slave management disabled"]
+      Enabled: [1, "Software slave management enabled"]
+    CPOL:
+      IdleLow: [0, "CK to 0 when idle"]
+      IdleHigh: [1, "CK to 1 when idle"]
+    CPHA:
+      FirstEdge: [0, "The first clock transition is the first data capture edge"]
+      SecondEdge: [1, "The second clock transition is the first data capture edge"]
+    LSBFRST:
+      MSBFirst: [0, "Data is transmitted/received with the MSB first"]
+      LSBFirst: [1, "Data is transmitted/received with the LSB first"]
+    MASTER:
+      Slave: [0, "Slave configuration"]
+      Master: [1, "Master configuration"]
+    SP:
+      Motorola: [0, "Motorola SPI protocol"]
+      TI: [1, "TI SPI protocol"]
+    COMM:
+      FullDuplex: [0, "Full duplex"]
+      Transmitter: [1, "Simplex transmitter only"]
+      Receiver: [2, "Simplex receiver only"]
+      HalfDuplex: [3, "Half duplex"]
+    IOSWP:
+      Disabled: [0, "MISO and MOSI not swapped"]
+      Enabled: [1, "MISO and MOSI swapped"]
+    MIDI: [0, 15]
+    MSSI: [0, 15]
+
+  IER:
+    TSERFIE:
+      Masked: [0, "TSER loaded interrupt masked"]
+      NotMasked: [1, "TSER loaded interrupt not masked"]
+    MODFIE:
+      Masked: [0, "Mode fault interrupt masked"]
+      NotMasked: [1, "Mode fault interrupt not masked"]
+    TIFREIE:
+      Masked: [0, "TI frame format error interrupt masked"]
+      NotMasked: [1, "TI frame format error interrupt not masked"]
+    CRCEIE:
+      Masked: [0, "CRC error interrupt masked"]
+      NotMasked: [1, "CRC error interrupt not masked"]
+    OVRIE:
+      Masked: [0, "Overrun interrupt masked"]
+      NotMasked: [1, "Overrun interrupt not masked"]
+    UDRIE:
+      Masked: [0, "Underrun interrupt masked"]
+      NotMasked: [1, "Underrun interrupt not masked"]
+    TXTFIE:
+      Masked: [0, "Transmission transfer filled interrupt masked"]
+      NotMasked: [1, "Transmission transfer filled interrupt not masked"]
+    EOTIE:
+      Masked: [0, "End-of-transfer interrupt masked"]
+      NotMasked: [1, "End-of-transfer interrupt not masked"]
+    DPXPIE:
```

Comment:
This matches the SVD but the RM says `DXPIE` which I think is correct; we should probably fix the SVD and update the entry here.

Reply:
ACK. Renamed.